app/assets/stylesheets/dm/custom_font.scss

```diff
@@ -24,3 +24,7 @@ i.dm {
 
   &.dm-saluting-veteran:before { content: "\e900";}
 }
+
+.font-normal {
+  font-weight: normal;
```

Comment:
you can probably use `.text-normal` from uswds: https://designsystem.digital.gov/utilities/text-styles/

Reply:
👍